pytype/pytd/pytd_visitors.py

```diff
@@ -694,7 +699,9 @@ def MaybeCapitalize(self, name):
   def VisitGenericType(self, node):
     """Convert a generic type to a string."""
     parameters = node.parameters
-    if self._NeedsTupleEllipsis(node):
+    if self._IsEmptyTuple(node):
+      return self.MaybeCapitalize(node.base_type) + "[()]"
```

Comment:
just set `parameters = "()"` here

Reply:
sorry, `parameters = ("()",)` since it needs to be a tuple

Reply:
Oh, yeah. I could have done that. 
I need to read a bit more and get familiar with the codebase.
Resolved.